src/Illuminate/Mail/Transport/MailgunTransport.php

```diff
@@ -71,12 +79,10 @@ public function stop()
 	 */
 	public function send(Swift_Mime_Message $message, &$failedRecipients = null)
 	{
-		$client = $this->getHttpClient();
-
-		return $client->post($this->url, ['auth' => ['api', $this->key],
-			'body' => [
-				'to' => $this->getTo($message),
-				'message' => new PostFile('message', (string) $message),
+		return $this->client->post($this->url, ['auth' => ['api', $this->key],
```

Comment:
You can check `$client::VERSION` here and either use the v5 or v6 args to support both versions.